frame/assets/src/migration.rs

```diff
@@ -44,10 +44,10 @@ pub mod v1 {
 			let status = if self.is_frozen { AssetStatus::Frozen } else { AssetStatus::Live };
 
 			AssetDetails {
-				owner: self.owner,
-				issuer: self.issuer,
-				admin: self.admin,
-				freezer: self.freezer,
+				owner: Some(self.owner),
+				issuer: Some(self.issuer),
+				admin: Some(self.admin),
+				freezer: Some(self.freezer),
```

Comment:
This is not `v1` though. I think [this](https://github.com/paritytech/substrate/blob/3e91823/frame/alliance/src/migration.rs#L26L42) is a better example for how to handle possible multiple migrations.